gdal/frmts/ecw/lookup.py

```diff
@@ -79,8 +79,8 @@ def load_dict( filename ):
         if len(tokens) < 3:
             continue
 
-	for i in range(len(tokens)):
-	    tokens[i] = string.strip(tokens[i])
+    for i in range(len(tokens)):
```

Comment:
this looks wrong. The previous indentation as in github UI looked better, but I see that there's a misinentation at line 85 / 86.